src/Backend/OnboardingInsuranceAPI/Program.cs

```diff
@@ -23,7 +27,8 @@ static async Task Main()
             {
                 services
                     .AddScopedByInterface<IHandler>()
-                    .AddDbContext<DataContext>(); //AddDbContextFactory try booth
+                    .AddCosmosDb(); //!!!PRODUCTION DB!!!
+                    //.AddInMemoryDb(); //For tests
```

Comment:
That is not how you do "test": https://docs.microsoft.com/en-us/visualstudio/test/walkthrough-creating-and-running-unit-tests-for-managed-code?view=vs-2022